main.js

```diff
@@ -541,6 +594,27 @@ function reportStatus() {
         eventLoopLags = [];
     }
 
+    states.setState(id + '.compactGroupProcesses',   {val: Object.keys(compactProcs).length, ack: true, from: id});
+    let realProcesses = 0;
+    let compactProcesses = 0;
+    Object.keys(procs).forEach(proc => {
+       if (procs[proc].process) {
+           if (procs[proc].startedInCompactMode) {
+               compactProcesses++
+           }
+           else {
+               realProcesses++;
+           }
+
+       }
+    });
+    states.setState(id + '.instancesAsProcess',   {val: realProcesses, ack: true, from: id});
+    states.setState(id + '.instancesAsCompact',   {val: compactProcesses, ack: true, from: id});
+
```

Comment:
Not too sure here... Is this supposed to be true?
`compactGroupProcesses = compactProcesses + realProcesses`
Because it might be that the else branch of line 601 is taken and then there would be a different count.

Reply:
compactGroupProcesses = "number of compact group controllern" (not adapter or instances!)